build/docs.js

```diff
@@ -8,6 +8,7 @@ const _ = require('lodash');
 const Bluebird = require('bluebird');
 const del = require('del');
 
+const argv = require('yargs').argv;
```

Comment:
Do we really need to add a whole dependency for just a single startup param? I'd actually rather have a string includes check, unless we plan on refactoring the current system to fully use yargs.